src/ontology/mondo-edit.obo

```diff
@@ -346705,6 +346709,9 @@ id: MONDO:0014973
 name: sudden cardiac failure, infantile
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
+synonym: "infantile sudden cardiac failure" EXACT [PMID:27523597, PMID:34400813, PMID:37869221]
+synonym: "PPA2 deficiency" EXACT [PMID:27523597, PMID:34400813, PMID:37869221]
```

Comment:
I made this one a broad synonym as it is possible that other PPA2 deficiency leads to other diseases. (unless PPA2 has been shown to related only to PPA2.